lib/controllers/posts.js

```diff
@@ -0,0 +1,21 @@
+const { Router } = require('express');
+const Post = require('../models/Post');
+const authenticate = require('../middleware/authenticate');
+
+module.exports = Router()
+  .get('/', authenticate, async (req, res, next) => {
+    try {
+      const data = await Post.getAll();
+      res.json(data);
+    } catch (e) {
+      next(e);
+    }
+  })
+  .post('/', async (req, res, next) => {
```

Comment:
this route also needs to be authenticated - to ensure only logged in users are able to post